readme.md

```diff
@@ -282,6 +282,7 @@ Other awesome open source projects that use Lighthouse.
 * **[lighthouse-lambda](https://github.com/joytocode/lighthouse-lambda)** - Run Lighthouse on AWS Lambda with prebuilt stable desktop Headless Chrome.
 * **[lighthouse-security](https://github.com/voorhoede/lighthouse-security#readme)** - Run a set of security audits along with Lighthouse.
 * **[Garie](https://github.com/boyney123/garie)** — An open source tool for monitoring performance using Lighthouse,  PageSpeed Insights, [Prometheus](https://prometheus.io/), [Grafana](https://grafana.com/) and [Docker](https://www.docker.com/).
+* **[lighthouse-ci](https://github.com/andreasonny83/lighthouse-ci)** - Integrate Google Lighthouse in your favourite Continuous Integration solution with zero configuration.
```

Comment:
i will admit we're working on making some CI stuff for lighthouse so we will likely ask you for this npm module name soon. :) (i've sent you some of our plans)  

wdyt about renaming? something like `lighthouse-score-assert` seems cool.

--------------------

and lastly an alternative implementation could be something like...

```
lighthouse http://mysite.com --output=json | lighthouse-score-assert --performance=90 --pwa=100
```

this way you don't have to support all of `lighthouse` CLI's flags in your own app, but can still support people who change throttling settings, pass in config-path's, etc

this seems mostly reasonable with your project except for the chromeFlags you add on. but that's not too bad for users. we plan to publish a docker container that will take care of all those sandboxing issues etc and make running lighthouse in CI a straightforward affair.